src/app/example-module/another-component/another-component.component.ts

```diff
@@ -7,10 +10,17 @@ import { Component, OnInit } from '@angular/core';
 })
 export class AnotherComponentComponent implements OnInit {
 
-  constructor() {
+  loginResult: LoginResult;
+
+  constructor(private loginService: LoginService) {
   }
 
   ngOnInit() {
-  }
+    const loginRequest: LoginRequest = {
+      username: 'username',
+      admin: true,
```

Comment:
Jak się zaktualizuje `papaya-interface`, to tu się biedak sypie

Reply:
co znaczy sypie?

Reply:
No tu nie ma tego admina już

Reply:
zmieniałaś ręcznie commit w submodule?

Reply:
Tak właśnie - jak dla mnie jeden aktualny master powinien być kompatybilny z drugim aktualnym masterem

Reply:
to nic dziwnego że się sypie, skoro pisałem to już jakiś czas temu
A co do zgodności masterów to się nie zgadzam - front korzysta z api w jakiejś wersji i taka jest pobierana w interface. Podbijając api niejako bierzesz na siebie odpowiedzialność za jego wprowadzenie.

Nie musi być przecież tak, że każda zmiana w api jest od razu wprowadzana też na front, bo bez niej front nie wstanie.

Reply:
>  front korzysta z api w jakiejś wersji i taka jest pobierana w interface

Ta wersja jest gdzieś określona czy to jest "wersja z wtedy, kiedy dana osoba ostatnio wpisała sobie `git submodule update`"?

Reply:
no jest w którymś z plików

Reply:
jak sobie wejdziesz z terminala do tego folderu, to jest tam detached head z commitem, który jest uznawany za ten 'wybrany'

Reply:
Aaa, dobra, to okej - ja się nie czepiałam dla zasady tylko bardziej z myślą o CI

Reply:
I tak bym głosowała za tym, żeby wywalić tego admina jeszcze tutaj, ale spoko

Reply:
jak czysty clone zrobi submodule update, to pobierze się zawsze ten poprawny